src/gsi/hybrid_ensemble_parameters.f90

```diff
@@ -428,6 +429,8 @@ module hybrid_ensemble_parameters
   real(r_kind),allocatable:: region_lat_ens(:,:),region_lon_ens(:,:)
   real(r_kind),allocatable:: region_dx_ens(:,:),region_dy_ens(:,:)
 
+  logical sete2s_flg
```

Comment:
This variable is also not used , either.

Reply:
Correct.  Code has been modified.